src/DateTimePicker.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+import Flatpickr from 'flatpickr';
+
+/**
+ * @module DateTimePicker
+ */
+class DateTimePicker extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || '',
+			isSupported: false
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	// test for browser support
+	componentWillMount() {
+		const input = document.createElement('input'),
+			invalidValue = 'notadate';
+
+		input.setAttribute('type', 'datetime-local');
+		input.setAttribute('value', invalidValue);
+
+		// adding false to view storybook
+		// in chrome since chrome supports
+		const isInvalidValueDenied = (false && input.value !== invalidValue);
```

Comment:
probably simpler/clearer to just call this `isSupported`.

Might be easier to have an optional boolean prop to force 'not supported' rather than hard-coding `false &&` in development.